RampUpAuto/src/main/java/seleautomation/com/RampUpAuto/GoogleSearch.java

```diff
@@ -0,0 +1,47 @@
+package Testing;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import java.util.concurrent.TimeUnit;
+
+   public class TestTest  {
+WebDriver driver;
+@BeforeMethod
```

Comment:
Try to use the annotation of ` @BeforeClass` instead of `@BeforeMethod` because we need to set up the webdriver one time for class level , this task contains just one test method but in case we have several methods then the code will continue to set up the driver before each method